cluster-autoscaler/simulator/cluster_snapshot_test.go

```diff
@@ -464,3 +463,132 @@ func TestNodeAlreadyExists(t *testing.T) {
 		}
 	}
 }
+
+func TestPVCUsedByPods(t *testing.T) {
+	node := BuildTestNode("node", 10, 10)
+	pod1 := BuildTestPod("pod1", 10, 10)
+	pod1.Spec.NodeName = node.Name
+	pod1.Spec.Volumes = []apiv1.Volume{
+		{
+			Name: "v1",
+			VolumeSource: apiv1.VolumeSource{
+				PersistentVolumeClaim: &apiv1.PersistentVolumeClaimVolumeSource{
+					ClaimName: "claim1",
+				},
+			},
+		},
+	}
+	pod2 := BuildTestPod("pod2", 10, 10)
+	pod2.Spec.NodeName = node.Name
+	pod2.Spec.Volumes = []apiv1.Volume{
+		{
+			Name: "v1",
+			VolumeSource: apiv1.VolumeSource{
+				PersistentVolumeClaim: &apiv1.PersistentVolumeClaimVolumeSource{
+					ClaimName: "claim1",
+				},
+			},
+		},
+		{
+			Name: "v1",
+			VolumeSource: apiv1.VolumeSource{
+				PersistentVolumeClaim: &apiv1.PersistentVolumeClaimVolumeSource{
+					ClaimName: "claim2",
+				},
+			},
+		},
+	}
+	nonPvcPod := BuildTestPod("pod1", 10, 10)
+	nonPvcPod.Spec.NodeName = node.Name
+	nonPvcPod.Spec.Volumes = []apiv1.Volume{
+		{
+			Name: "v1",
+			VolumeSource: apiv1.VolumeSource{
+				NFS: &apiv1.NFSVolumeSource{
+					Server:   "",
+					Path:     "",
+					ReadOnly: false,
+				},
+			},
+		},
+	}
+	testcase := []struct {
+		desc                 string
+		node                 *apiv1.Node
+		pods                 []*apiv1.Pod
+		claimName            string
+		exists               bool
+		isRemoveCheck        bool
+		removePod            string
+		claimNameAfterRemove string
```

Comment:
Why not just reuse `claimName`? It doesn't make sense to check different claims before and after pod removal.

Reply:
Done.